requirements/requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 black==22.3.0
 coverage
 dm-haiku==0.0.5
-flake8
```

Comment:
Shouldn't it be `4.0.1` in order to be consistent with the version in pre-commit yaml?